packages/commerce-sdk-react/src/constant.ts

```diff
@@ -9,7 +9,7 @@
  * This list contains domains that can host code in iframe
  */
 export const IFRAME_HOST_ALLOW_LIST = Object.freeze([
-    'runtime.commercecloud.com',
-    'runtime-admin-staging.mobify-storefront.com',
-    'runtime-admin-preview.mobify-storefront.com'
+    'https://runtime.commercecloud.com',
+    'https://runtime-admin-staging.mobify-storefront.com',
+    'https://runtime-admin-preview.mobify-storefront.com'
```

Comment:
@wjhsf I noticed that you did some changes with the allow list stuff. When I was testing my work after updating the code preview wouldn't load, I traced it down to the parent origin being passed in with the protocol and the allow list not having them. Let me know if this is an executable solution, or if you were intending on having it work agnostic to the protocol

Reply:
Nah, Runtime Admin is only ever `https`, so this is fine.